contrib/build-push-ceph-container-imgs.sh

```diff
@@ -55,11 +55,23 @@ CN_RELEASE="v2.3.1"
 
 function _centos_release {
   local release=$1
-  if [[ "${release}" =~ master|octopus|^wip* ]]; then
-    echo 8
-  else
-    echo 7
-  fi
+  case  "${release}" in
+    *mimic*)
+      echo 7
+      ;;
+    *nautilus*)
+      echo 7
+      ;;
+    *octopus*)
+      echo 8
+      ;;
+    master)
+      echo 8
+      ;;
+    *)
+      echo 8
+      ;;
+  esac
```

Comment:
```suggestion
  case  "${release}" in
    *mimic*)
      echo 7
      ;;
    *nautilus*)
      echo 7
      ;;
    *)
      echo 8
      ;;
  esac
```

should work as well. I don't think centos 9 will be available before the EOL of nautilus.

Reply:
yeah, it is simpler. i wanted to be explicit.

Reply:
@dsavineau in comparison with @sebastian-philipp version, i'd prefer his alternative. as a "case" clause is easier to read in this case, IMHO.